api/sds.proto

```diff
@@ -93,6 +93,9 @@ message CertificateValidationContext {
 
   // Must present signed certificate time-stamp.
   google.protobuf.BoolValue require_signed_certificate_timestamp = 6;
+
+  // If specified, Envoy will verify certificates against this CRL.
+  DataSource crl = 7;
```

Comment:
Per your other PR in which you mention having multiple CRLs, should this be repeated? (From an API perspective please future-proof it).